aws-ts-vpc-with-ecs-fargate-py/ecs-fargate-python/README.md

```diff
@@ -0,0 +1,122 @@
+
+### What Is This?
+
+This example shows authoring Infrastructure as Code in `python` for ecs fargate with a [crosswalk vpc](https://www.pulumi.com/docs/guides/crosswalk/aws/vpc/) built in `typescript`.  It
+provisions a full [Amazon Elastic Container Service (ECS) "Fargate"](https://aws.amazon.com/ecs) cluster and
+related infrastructure, running a load-balanced NGINX web server accessible over the Internet on port 80.
+This example is inspired by [Docker's Getting Started Tutorial](https://docs.docker.com/get-started/).
+
+### Why would you do this?  
+An example showing that you can easily infrastructure written in a different language than the one you are used to.  The vpc outputs from vpc-crosswalk-ts folder are used as inputs via [StackReference](https://www.pulumi.com/docs/intro/concepts/organizing-stacks-projects/#inter-stack-dependencies)
+
+## Prerequisites
+
+* [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+* [Configure Pulumi to Use AWS](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/) (if your AWS CLI is configured, no further changes are required)
+
+
+### Mandatory AWS Pre-REQ: AWS Console Fix for Tags:
+
+This is necessary so that the tags work properly in ecs
+[Tagging your Amazon ECS resources](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-using-tags.html)
+
+As per AWS:  `You must opt in to the new Amazon Resource Name (ARN) and resource identifier (ID) formats.`
+
+This has to be done per region until AWS enables it as default across the board(April 1, 2021).
+
+AWS Console -> Elastic Container Service ->  Account Settings -> 
+
+BEFORE
+```
+Resource                My IAM user or role account settings 
+Container Instance      Undefined
+Service                 Undefined
+Task                    Undefined
+```
+
+AFTER
+```
+Resource                My IAM user or role account settings 
+Container Instance      Enabled
+Service                 Enabled
+Task                    Enabled
+```
+
+## Running the Example
+
+ 1. Initialize a new stack called: `ecs-fargate-dev` via [pulumi config](https://www.pulumi.com/docs/reference/cli/pulumi_config_set/). 
+      ```
+      $ pulumi stack init ecs-fargate-dev
+      ```
+
+2. Create a Python virtualenv, activate it, and install dependencies:
+
+    This installs the dependent packages [needed](https://www.pulumi.com/docs/intro/concepts/how-pulumi-works/) for our Pulumi program.
+
+    ```bash
+    $ python3 -m venv venv
+    $ venv/bin/python -m pip install --upgrade pip setuptools wheel
+    $ venv/bin/python -m pip install -r requirements.txt
+    ```
+
+2. View the current config settings. This will be empty.
+   ```
+   $ pulumi config
+   ```
+   ```
+   KEY                     VALUE
+   ```
+
+4. Populate the config.
+
+   Here are aws [endpoints](https://docs.aws.amazon.com/general/latest/gr/rande.html)
+
+   Note: the mystackpath is the key.  The value for it will be your `stackreference` from the vpc:
```

Comment:
If possible, can you just add a comment that the stackreference format is org/project/stack as people may be confused. It's also worth noting that this is the format ONLY for a SaaS based backend

Reply:
Added this information